include/openenclave/bits/eeid.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/**
+ * @file eeid.h
+ *
+ * This file defines the EEID structure.
+ *
+ */
+
+#ifndef _OE_BITS_EEID_H
+#define _OE_BITS_EEID_H
+
+#include "properties.h"
+
+OE_EXTERNC_BEGIN
+
+/*
+**==============================================================================
+**
+** oe_eeid_t
+**
+**==============================================================================
+*/
+typedef struct oe_eeid_t_
+{
+    uint32_t hash_state_H[8];
+    uint32_t hash_state_N[2];
```

Comment:
The above field names aren't self-explanatory.  Please add comments explaining what they are.